rdkafka-sys/build.rs

```diff
@@ -292,7 +292,7 @@ fn build_librdkafka() {
     }
 
     if !cmake_library_paths.is_empty() {
-        env::set_var("CMAKE_LIBRARY_PATH", cmake_library_paths.join(";"));
+            env::set_var("CMAKE_LIBRARY_PATH", cmake_library_paths.join(";"));
```

Comment:
This is making the linter unhappy. Pls see https://github.com/fede1024/rust-rdkafka/actions/runs/15254430333/job/43060466022#step:4:35